README.md

```diff
@@ -63,3 +63,21 @@ The charts in the `Incubator` directory allows charts to be shared and improved
 ## Status of the Project
 
 This project is still under active development. If you experience any [issues](https://github.com/nuodb/nuodb-helm-charts/issues) please let us know, or better yet, contribute a fix or feature.
+
+## Testing
+
+To run the tests in this project, Go 1.12 is required.
+The easiest way to clone the repository and run a test suite (e.g. `test/integration`) is as follows:
+
+```
+GO111MODULE=off go get github.com/nuodb/nuodb-helm-charts
+cd $(go env GOPATH)/src/github.com/nuodb/nuodb-helm-charts
```

Comment:
I am not sure whether `go get` gets the newest. You can simply clone the repo to any location. It doesnt have to be in GOPATH.

Reply:
`GO111MODULE=off go get` does seem to get the newest. The reason for using `go get` instead of `git clone` is that I ended up having to put my clone of the repo into the `GOPATH` to get my IDE to do static analysis and navigation (I use Spacemacs with the Go layer).

Reply:
Yeah. It was the case with old Go :) I can't speak for spacemacs.